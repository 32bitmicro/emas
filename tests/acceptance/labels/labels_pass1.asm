label_0:
.word label_0
label_1:
nop nop nop nop nop nop nop nop nop nop
label_2:
.word label_0, label_1, label_2
