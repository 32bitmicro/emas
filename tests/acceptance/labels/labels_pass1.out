@ 0x0000 : 0x0000  /  000 000 0 000 000 000  /  0
@ 0x0001 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0002 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0003 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0004 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0005 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0006 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0007 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0008 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x0009 : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x000a : 0xe000  /  111 000 0 000 000 000  /  57344
@ 0x000b : 0x0000  /  000 000 0 000 000 000  /  0
@ 0x000c : 0x0001  /  000 000 0 000 000 001  /  1
@ 0x000d : 0x000b  /  000 000 0 000 001 011  /  11
